HW05_part_B.Rmd

```diff
@@ -0,0 +1,173 @@
+---
+title: "HW05 Part B"
+author: "Patrick Haller"
+date: "7/31/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## HW05 Part B
+
+
+```{r load packages, include = FALSE}
+
+library(tidyverse)
+
+```
+
+**First, I want to read in the data from the files in the CTRP_files folder and create a new data frame that combines all of the files initially given to us.**
+
+```{r read in data}
+
+# read in the data files from the CTRP_files folder
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+cell_line_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+compound_info <- read.csv("CTRP_files/compound_info.csv")
+experiment_info <- read.csv("CTRP_files/experiment_info.csv")
+
+
+```
+
+
+
+```{r}
+
+# generate new data frame that combines the 5 initial data frames
+data_complete <- AUC_1 %>%
+  # combine data rows from AUC_2 data frame
+  bind_rows(AUC_2) %>%
+  # join the columns from the experiment info data frame by experiment_id
+  inner_join(experiment_info, by = c("experiment_id" = "expt_id")) %>%
+  # join the columns from the cell line info data frame by master_ccl_id
+  inner_join(cell_line_info, by = "master_ccl_id") %>%
+  # join the columns from the compound info data frame by master_cpd_id
+  inner_join(compound_info, by = "master_cpd_id")
+  
+
+  
+```
+
+
+**With this new data frame, we can answer the following questions:**  
+
+### Which cancer type has the lowest AUC values to the compound "vorinostat"?
+
+```{r}
+
+# create new data frame for this problem
+auc_low_vor <- data_complete %>%
+  select(area_under_curve, cancer_type, cpd_name) %>%
+  filter(cpd_name == "vorinostat", cancer_type != "NA") %>%
```

Comment:
I tried to use `cancer_type != NA` to filter observations but it returned no records, so I finally used `drop_na(cancer_type)`. Now I know that it was because that I should use character `"NA"` rather than logical constant `NA`.